.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
```

Comment:
What's this file for? Does gpuCI not cover something?

Reply:
I think this may be leftover from before gpuCI was added. Could probably be removed now.

Reply:
Actually I think this was created by the Jupyter Lab migration tool. So can definitely be deleted. But we may need to update gpuCI to match the functionality.